apps/vr-tests-react-components/src/stories/Popover.stories.tsx

```diff
@@ -89,6 +89,35 @@ storiesOf('Popover Converged', module)
         </PopoverSurface>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29425&commitId=c2b1b732bd58ba088e379bf8eda3ad11b7772b41&env=prod&clientType=FLUENTUI)


  <details><summary><b>Popover Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.when rendering inline, it should not render behind relatively positioned elements.PopoverSurface focused.chromium.png|0| Added|
|Popover Converged.when rendering inline, it should not render behind relatively positioned elements.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>